brainrender_napari/napari_atlas_representation.py

```diff
@@ -15,23 +18,40 @@ class NapariAtlasRepresentation:
     mesh_opacity: float = 0.4
     mesh_blending: str = "translucent_no_depth"
 
+    def __post_init__(self) -> None:
+        """Setup a custom QLabel tooltip and enable napari layer tooltips"""
+        self._tooltip = QLabel(self.viewer.window.qt_viewer.parent())
```

Comment:
This line produces a deprecation warning:
```
FutureWarning: Public access to Window.qt_viewer is deprecated and will be removed in
v0.5.0. It is considered an "implementation detail" of the napari
application, not part of the napari viewer model. If your use case
requires access to qt_viewer, please open an issue to discuss.!
```

Reply:
Yes - I don't think we have a nice way around this if we want a tooltip until napari allows custom layer tooltips. It's something that should be easy to contribute to napari before 0.5.0 and then fix here, but better to get the tool to users earlier, I think?

Reply:
Agreed, but is there an ETA for v0.5.0? If it's soon, then we'll have an issue.

Reply:
from a zulip convo in June, it's not soon.